.github/workflows/aap_chatbot_build_and_deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: AAP Chatbot Stack - Build and Deploy
+
+on:
+  workflow_dispatch:
+
+jobs:
```

Comment:
It might be nice that when building _our_ stack it also ran the `llama_chatbot_build_and_deploy.yml` workflow?

This can be accomplished with the `uses` keyword; [for example](https://github.com/ansible/ansible-ai-connect-operator/blob/main/.github/workflows/pr-build-and-publish-artifacts.yaml#L10-L12).

WDYT?

Reply:
I have been able to use a local docker registry for GH actions finally, so everything now is done in a single workflow. Easier, nicer and works well. :)